state/block.go

```diff
@@ -15,6 +16,10 @@ type Block struct {
 	Checked     bool
 }
 
+func (b *Block) String() string {
+	return fmt.Sprintf("BlockNumber: %d, BlockHash: %s, ParentHash: %s, ReceivedAt: %s", b.BlockNumber, b.BlockHash, b.ParentHash, b.ReceivedAt)
+}
+
```

Comment:
I think this is unused code. please add it when it is necessary

Reply:
This code is used in `synchronizer.go:854`